contentctl/objects/config.py

```diff
@@ -249,11 +254,52 @@ class StackType(StrEnum):
     classic = auto()
     victoria = auto()
 
+
 class inspect(build):
-    splunk_api_username: str = Field(description="Splunk API username used for running appinspect.")
-    splunk_api_password: str = Field(exclude=True, description="Splunk API password used for running appinspect.")
+    # TODO (cmcginley): can we enable enrichments by default for the inspect command?
+    splunk_api_username: str = Field(
```

Comment:
Note that in practice, not all users are created equal wrt to access to both Splunkbase AND the Appinspect API; the Splunkbase user/pass we use in SCA did not seem to have privileges for Appinspect, but the Appinspect user/pass used in SC had access to both Splunkbase and Appinspect. The need for both sets of permissions should probably be highlighted in documentation somewhere

Reply:
Honestly no clue. The provisioning and management of those accounts is all a blackbox to me. This observation of mine is just anecdotal, not ground truth.